bgbb/sql/sql_utils.py

```diff
@@ -0,0 +1,219 @@
+import datetime as dt
+from typing import List
+
+import pandas as pd
+
+from bgbb.sql.sql_utils_tests import test_ho_range, test_model_range, test_rft
```

Comment:
The tests should be separate from the application code, unless you want to assert properties about the incoming data.

Reply:
👍